### ISSUE TITLE
The future of topic card contents... (Links, Markdown, Embedly)

### ISSUE THREAD

**Issue by Bortseb** [metamaps/metamaps#790]
Initially, I just wanted to create an issue simply to say that sometimes embedly takes a reaaaally long time to produce something that I can interact with. For certain use-cases of metamaps this is pretty glaring flaw.

One way I would like to use metamaps is simply as a non-linear & visual bookmarking tool.  So if I am just quickly trying to go to some of my favorite links, having to double tick a topic to open it is already an added step in getting to my desired link... Having to wait for embedly's loading symbol to close and show me something I can interact with makes this use-case for MM almost too annoying to bother. 

Can we include embedly inside of markdown? Instead of allocating a single link for each topic that is given supreme importance? Can we show a link that can be interacted while embedly is loading? 

I think this second suggestion is a good short-term solution, until a longer term design comes about... this way it doesn't matter if embedly is taking a long time to load... I can still get at my link immediately, regardless of what embedly is doing.

**Comment by devvmh** [comment]
Hmm. It is kind of weird for me because even once embedly loads ive never found it too useful

I hesitate to allow too much embedding in topic cards because if cross site scripting. Ive got markdown in safe mode, so it doesnt allow too much arbitrary html.

What are some use cases you've wanted to use embedding for? I think that really would help frame the problem for me personally

**Comment by Bortseb** [comment]
Actually, my emphasis here wasn't intended to be on embedding content... I just figured since we already have embedly I wasn't going to suggest we drop support for this feature...

I'd be fine if all we had was markdown and no more embedly actually... though... it would be really neat to still embed certain things like youtube... (I just figured maybe it wouldn't be all that hard to allow embedly stuff within markdown?)... 

having a special "Link" field seems a little strange now with markdown support coming on board,  where you can have as many links as you like...

Basically, I just want to open the card and be able to immediately interact with my links inside, whether we keep supporting embedly or not (i.e in both cases)

**Comment by devvmh** [comment]
What youre describing  is already possible on the latest develop isn't it?

I've been thinking the ultimate setup is markdown in the desc field, and
then one field for an embedded link and another for attached files.

Embedly being  slow is annoying. it would be great to writr code that kept
the iframe alive in the background so subsequent loads were instant

**Comment by Connoropolous** [comment]
opened this here instead: https://trello.com/c/wrxoX4kQ/31-one-way-i-would-like-to-use-metamaps-is-simply-as-a-non-linear-visual-bookmarking-tool
